iothub/service/src/Registry/Models/IotHubJobResponse.cs

```diff
@@ -0,0 +1,67 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices
+{
+    /// <summary>
+    /// Contains the properties set by the service import/export job.
+    /// </summary>
+    public abstract class IotHubJobResponse
+    {
+        /// <summary>
+        /// Creates an instance of this class. Provided for unit testing purposes only.
```

Comment:
This is provided for serialization, not just unit testing.

Also, should it be protected internal?